### DISCUSSION TITLE
Does znap install work for you?

### DISCUSSION THREAD

**Discussion by LoGaIta99** [marlonrichert/zsh-autocomplete#584] [General]
After updating with znap pull today zsh-autocomplete broke. Uninstalling and reinstalling with znap did not work. It appears that a wrong link is created in ~/.local/state, pointing to the file in the cloned repo instead of the folder in ~/.local/share. To make things work again I needed to use the manual installation method.
Have someone had the same issue? I'm on openSUSE linux.
This is particularity strange because znap plugin manager is from the same author, but still, this plug-in is the only one which broke with znap pull.

**Comment by marlonrichert** [comment]
This should be fixed now on the `main` branch.
